python/paddle/fluid/layers/nn.py

```diff
@@ -9099,6 +9096,9 @@ def affine_grid(theta, out_shape, name=None):
                                              ``out_shape`` can be a Tensor or a list or tuple. The data
                                              type must be int32.
         name(str|None): The default value is None.  Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name`.
+        align_corners(bool): Whether to align corners of target feature map and source feature map. Default: True.
+        use_cudnn(bool): It will ignore `align_corners` and compute in align corners mode when use_cudnn is true.
```

Comment:
remove `use_cudnn` from API
- check cudnn availability with #26180
- `align_corners` is false, set `use_cudnn` attr to false